api-specificatie/openapi.yaml

```diff
@@ -86,7 +83,77 @@ paths:
+        '429':
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/429'
+        '500':
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/500'
+        '503':
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/503'
+        default:
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/default'
+
+  /adressen:
+    get:
+      security:
+        - apiKeyBAG: []
+      operationId: raadpleegAdresMetKeuzeItemId
+      tags:
+        - Adres
+      summary: Raadpleeg adres met de identificatie van het geselecteerde item 
+      description: "Raadpleeg het adres met de identificatie van het geselecteerde item verkregen via de zoeken-operatie
+        Geleverd worden de huidige gegevens van een adres evt. met embedded woonplaats, openbare ruimte, nummeraanduiding. 
+        Huidige gegevens zijn actuele gegevens die geen eind status hebben, zoals 'Naamgeving ingetrokken'. 
+        Als de nummeraanduiding is ingetrokken, dan levert een bevraging geen adres op. 
+        Relaties met adresseerbaarobject (ligplaats, standplaats of verblijfsobject) en als het adresseerbaare object een verblijfsobject betreft het/de gerelateerde pand(en). 
+        Parameter keuzeItemIdentificatie bevat identificatie van een keuze item verkregen uit een zoek-operatie.
+        Parameter adresString bevat een weergavenaam zoals verkregen uit een zoek-operatie.
+        Parameter adresseerbaarobjectIdentificatie bevat de 16 cijferige identificatie van een ligplaats, standplaats of verblijfsobject."
+      parameters:
+        - $ref: '#/components/parameters/keuzeItemIdentificatie'
+        - $ref: '#/components/parameters/adresString'
+        - $ref: '#/components/parameters/adresseerbaarObjectIdentificatie-query'
+        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/api-version"
+        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/expand"
+        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/fields"
+      responses:
+        '200':
+          description: Bevraging raadpleegAdresMetKeuzeItemId geslaagd
+          headers:
+            api-version:
```

Comment:
Graag Haal Centraal common yaml gebruiken voor de headers.

Reply:
Ik heb issue #59 aangemaakt omdat niet alle benodigde headers in de common.yaml zijn gedefinieerd.